drivers/sub8_videoray_m5_thruster/config/firmware_settings/readme.md

```diff
@@ -0,0 +1,7 @@
+# VideoRay Thruster Firmware Configuration
+This directory contains the firmware configurations meant to be used with our VideoRay m5 thrusters.
```

Comment:
If the documentation is external, it should probably be linked or referred to in the readme.